tests/assertions.py

```diff
@@ -52,7 +52,7 @@ def assert_dataarray_equal(
 
     # if decimals are supplied, do a rounding, otherwise rounding is just a dummy 'identity' func
     # pylint: disable=unnecessary-lambda-assignment
-    rounding = lambda x: x if decimals is None else np.round(x, decimals=decimals)
+    rounding = lambda x: x if decimals is None else np.round(x, decimals=decimals)  # noqa: E731
```

Comment:
I think we should delete this function and just use the xr.testing functions instead.

Reply:
I'll raise a new ticket for this, is it would change actual logic, so I think it's best to be cautious.